docs/release_notes.md

```diff
@@ -14,7 +14,9 @@ The following changes are not backward compatible with the previous release.
 
 ### Deprecation notes
 
-None
+The following capabilities have been deprecated and cannot be used.
+
+- The `osm_injector_injector_sidecar_count` and `osm_injector_injector_rq_time` metrics have been removed. The `osm_admission_webhook_response_total` and `osm_http_response_duration` metrics should be used instead.
```

Comment:
With this change, the `osm_injector_injector_rq_time` and `osm_injector_injector_sidecar_count` metrics can be removed from the OSM Mesh and Envoy Details Grafana dashboard.